spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -167,7 +167,7 @@ def paginated_collection(*args)
       end
 
       it "should display 'No entries found'" do
-        expect(pagination.find_by_class('pagination_information').first.content).to eq "No entries found"
+        expect(pagination.find_by_class('pagination-information').first.content).to eq "No entries found"
```

Comment:
Line is too long. [105/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.